helpers/image_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'tempfile'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    Tempfile.create('images-', StaticData::TMP_FOLDER.to_s) do |fo|
+      fo.binmode
+      fo.write(URI.parse(url).open(&:read))
+      fo.rewind
```

Comment:
I don't think rewind is needed in that context

Reply:
yes, there is no need to use rewind